lib/ansible/modules/network/eos/eos_vlans.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for eos_vlans
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+}
+
+DOCUMENTATION = """
+---
+module: eos_vlans
+version_added: 2.9
+short_description: Manage VLANs on Arista EOS devices.
+description: This module provides declarative management of VLANs on Arista EOS network devices.
+author: Nathaniel Case (@qalthos)
+notes:
+- Tested against Arista EOS 4.20.10M
```

Comment:
Please add the connection type and link to the platform guide. Since extends_documentation_fragment is not used in resource module documentation it needs to be added explicitly.